### PR TITLE
Merge branch 'master' of https://github.com/rusefi/rusefi

### PR DESCRIPTION
**Pull request by AlexHLinS**

### CODE REVIEW

**Comment by rusefillc** [comment]
Note 'three files changes' in https://github.com/rusefi/rusefi/pull/2236/files

I assume that's not intended since only one file related to the change needed?

to make things simplest I can cherry pick just the change and push under my name - or please remove unneeded changes from this PR or re-create PR with just one file changed? your call.

File: firmware/config/engines/vw_b6.cpp [hunk 1]
```diff
@@ -25,16 +25,17 @@ void setVwPassatB6(DECLARE_CONFIG_PARAMETER_SIGNATURE) {
...
+	// seting Injectors flow rate equal 1214 cc/min
+	engineConfiguration->injector.flow = 1214;
+	// seting pressure to high
+	engineConfiguration->injectorPressureType = IPT_High;
+	// setting to Sensed Rail Pressure mode
+	engineConfiguration->injectorCompensationMode = ICM_SensedRailPressure;
+	// setting Fuel rail pressure to 10 000 kPa
+	engineConfiguration->fuelRailPressure = 10000;
+	//setting "flat" 0.2 ms injector's lag time
+	setArrayValues(engineConfiguration->injector.battLagCorr, 0.2);
```
Comment (AlexHLinS) [inline firmware/config/engines/vw_b6.cpp:37]: This part is resolving the ticket.

**Comment by AlexHLinS** [comment]
Done (28-37 lines of the vw_b6.cpp code have to be checked and applyed)

**Review by AlexHLinS — COMMENTED** [review]
This part of the code was edited.

**Comment by rusefillc** [comment]
Thank you! I've manually cherry picked the b6 change.

**Pull request closed without merging**